Makefile

```diff
@@ -1,10 +1,20 @@
 PACKAGES=$(shell go list ./... | grep -v 'tests')
-HERUMI= $(shell pwd)/.herumi
-BLS_CGO_LDFLAGS=CGO_LDFLAGS="-L$(HERUMI)/bls/lib -lbls384_256 -lm -lstdc++"
-BUILD_LDFLAGS= -ldflags "-X github.com/zarbchain/zarb-go/version.build=`git rev-parse --short=8 HEAD`"
 
+TAGS="zarb"
+VTAG=$(shell git describe --tags --abbrev=0)
+HERUMI= .herumi
+LDFLAGS= -ldflags "-X github.com/zarbchain/zarb-go/version.build=`git rev-parse --short=8 HEAD` -X github.com/zarbchain/zarb-go/version.semVer=$(VTAG)"
 
 
+_OS=$(shell go env GOOS)
+_ARCH=$(shell go env GOARCH)
+
+
+LIB_DIR=$(shell pwd)/$(HERUMI)/bls/lib/$(_OS)/$(_ARCH)
+CGO_LDFLAGS=CGO_LDFLAGS="-L$(LIB_DIR) -lbls384_256 -lm -lstdc++"
+CGO_LDFLAGS_Default=$(shell go env CGO_CFLAGS)
```

Comment:
`CGO_LDFLAGS_Default=$(shell go env CGO_LDFLAGS)` is better.
`CFLags` mostly uses for compiling and `LDFLAGS` for linking. A bit different.

Reply:
ohh shoot🤦‍♂️
absolutely right